### DISCUSSION TITLE
Add support for going to the next unread not-muted chat

### DISCUSSION THREAD

**Discussion by mrosso10** [d99kris/nchat#267] [Ideas]
Hi everyone, and thanks Kris for this (can't find yet the right adjective, but will go with -enjoyable-) piece of software.

I was thinking about a feature to cycle through the unread chat list (like Ctrl+F), but excluding the muted ones.

What do you think?

**Comment by d99kris** [comment]
Hi @mrosso10 - thanks for the positive feedback 👍 

Have you seen the `ui.conf` option [muted_indicate_unread](https://github.com/d99kris/nchat#muted_indicate_unread)? 

> Specifies whether chat list should indicate unread status * for muted chats. This also determines whether the such chats are included in jump to unread.

Would this cover your use case (set `muted_indicate_unread=0` to try)? 

Btw, on a related note when I tested `muted_indicate_unread=0` just now, I found a bug that muted chats with **cached** unread messages may wrongly be shown as unread (`*` after name in chat list), and thus also included with Ctrl-F jump to next chat. I'm preparing and testing a fix for that.

**Reply by d99kris** [reply]
> I'm preparing and testing a fix for that.

FYI I've fixed this bug now in 846bac8 / v5.1.16.

**Reply by mrosso10** [reply]
It did the trick! Thank you very much for that bugfix :raised_hands:
